setup.cfg

```
@@ -0,0 +1,45 @@
+[metadata]
+name = copyist
+version = 0.1.dev0
+author = Polyconseil
+author_email = opensource+copyist@polyconseil.fr
+description = A toolkit for developers
```

A bit vague, perhaps?

```suggestion
description = A tool to keep your configuration synced between repositories
```